### DISCUSSION TITLE
Como ingresar estados de inputs y mostrarlo en pagina de Pastelero. VUEX

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#486] [Q&A] [answered]
Aquí nuevamente molestándolos Ya me quebré la cabeza, no se si hay algo que necesite tener en cuenta.

Tengo entendido que hay que guardar en el estado los parámetros de el formulario. Pero la verdad al ingresar los datos y darle click en el botón submit, no me deja guardados los datos.

Alguno de ustedes que ya lo haya hecho podría apoyarme solo para darme una guía de lo que pudiera hacer?

**Comment by gersondlarosa** [comment]
Estas haciendo la practica de la pastelería ?? o que es lo que necesitas no soy muy bueno pero buscándole te puedo apoyar

**Reply by OmarVenturaP** [reply]
Así es, pero ya vez que la primera práctica hicimos lo mismo pero de manera estática. Aquí el asunto es que supuse que iba a poder guardarse para de esa forma hacerlo dinámico. Pero creo que va a ser únicamente por el back. Gracias

**Comment by gersondlarosa** [comment]
A ya se mira eso ya se ve en el Back si quieres que se guarde deja los en duro para que sea una simulación por que para cachar los datos debes de tener una base de datos y de ahí extraer la info

**Comment by doguedogue** [comment]
Al igual que en el contador los datos de pantalla pueden estar unidos a tus estados usando el v-bind o :.
En el HTML template
<Input v-bind="Pedido"...

En el script usas lo que tengas en
Data(): {
      Return {
           Pedido: null,
           Ingredientes: null,
           Etc: null
      }
},
Methods: {
       //Aquí lo insertas si quieres usar un método en el botón submit (pero ya está unido con v-bind)
       //this.Pedido = 123
}

**Reply by OmarVenturaP** [reply]
No se si sea igual, porque estamos usando VUEX y con eso se ocupa estados. Methods no se donde aplicaria. O al menos no se si se puede hacer. Ya vez que dijo el MC que era casi casi parte de la práctica los estados.

De hecho la primera practica lo hice asi. Pero en la misma página.

Aquí al parecer se requiere que se muestre en la página del pastelero y eso me quiebra la cabeza.

**Comment by OmarVenturaP** [comment, marked as answer]
Gracias por sus comentarios compañeros. Ya lo solucioné, tuve que instalar un plugin de Vuex para poder mandarlo a local Storage de allí poder jalarlo en la página de pastelero.
<b> Es demasiado fácil</b>, pero para poder encontrarlo tuve que quebrarme la cabeza por varias horas.

**Reply by jricardocam** [reply]
Omar podrías compartirme el plugin, aún no encuentro la solución :'(

**Reply by OmarVenturaP** [reply]
Te envio el link de donde obtuve la información. Ya la adaptas a tu código.

<a href="https://github.com/pinguinjkeke/vue-local-storage">Este es para si usas VUE</a>

<a href="https://github.com/championswimmer/vuex-persist">Este es para si usas VUEX</a>
